### ISSUE TITLE
References to `systemctl` during boot

### ISSUE THREAD

**Issue by dslm4515** [dslm4515/CMLFS#100]
When booting, these lines show up:

```
vtcon0: Process '/usr/bin/systemctl --no-block restart systemd-vconsole-setup.service' failed with exit code 1.
vtcon1: Process '/usr/bin/systemctl --no-block restart systemd-vconsole-setup.service' failed with exit code 1.
```

Perhaps a udev rule(s) were not patched?

**Comment by dslm4515** [comment]
Offending udev rule:

`/usr/lib/udev/rules.d/90-vconsole.rules`

**Comment by dslm4515** [comment]
fixed in commit 36b9e7bf14bc137b3be302cdfcde1815edad7cb8
